Dockerfile

```diff
@@ -5,8 +5,8 @@ RUN apt-get upgrade -y
 RUN apt-get install -yq git gcc make build-essential python3-dev python3-reportlab
 
 RUN mkdir -p /app/logs
-RUN git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
+
+RUN pip3 install --no-cache-dir -U -r requirements.txt
```

Comment:
Doesn't actually run if you build the docker container :)

This line is installing the `requirements.txt` that's supplied by the base container; not the `requirements.txt` of the app - the `requirements.txt` needs to be copied in.

However, even with that it doesn't seem to be compiling?